packages/@glimmer/destroyable/index.ts

```diff
@@ -2,11 +2,11 @@ import { scheduleDestroy, scheduleDestroyed } from '@glimmer/global-context';
 import type { Destroyable, Destructor } from '@glimmer/interfaces';
 import { debugToString } from '@glimmer/util';
 
-const enum DestroyingState {
```

Comment:
getting rid of enums is so nice!!! :tada: 
can't wait to see how this affects perf in a consuming project 

(enums are bad for perf, and compiled to IIFEs by default :sweat_smile: )